wallet/walletrpc.c

```diff
@@ -460,8 +460,8 @@ static struct command_result *json_listfunds(struct command *cmd,
 		json_object_start(response, NULL);
 		json_add_txid(response, "txid", &utxos[i]->txid);
 		json_add_num(response, "output", utxos[i]->outnum);
-		json_add_amount_sat(response, utxos[i]->amount,
-				    "value", "amount_msat");
+		json_add_amount_sat_compat(response, utxos[i]->amount,
+					   "value", "amount_msat");
```

Comment:
Maybe I'm off here, but isn't that supposte to be called `amount_sat`?

Reply:
No... we always print amounts as msat now, to avoid confusion.